src/SDKs/StreamAnalytics/StreamAnalytics.Tests/ScenarioTests/TransformationTests.ScenarioTests.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Azure.Management.Resources.Models;
```

Comment:
@shahabhijeet I manually added the 1.6.0 NuGet to our test project, updated the tests, re-ran them, and updated the session records.

Let me know if I did anything incorrectly. :)